### ISSUE TITLE
[Taxon]: MassFlowRate.Gas

### ISSUE THREAD

**Issue by MarkKuster** [NCSLI-MII/measurand-taxonomy#2]
### Contact Details

mjk@ieee.org

### What is the process type?

Source

### Measurand

This process sources a reference mass flow rate of gas for calibrating gas flow meters. The instantaneous mass flow rate $q_m$ equals $dm/dt$, sometimes estimated as $\Delta m / \Delta t$ using the total mass $\Delta m$ flowing through a defined space in time $\Delta t$.

### Kind of quantity

mass flow rate

### Does the measurand require a special token?

None

### Taxon Name

Source.MassFlowRate.Gas
A parameter handles the gas species and the CMC template probably involves many other parameters.

**Comment by ryanmwhitephd** [comment]
Yes, a number of parameters should be included. A discussion how to encode the parameters has been opened, see #52 
Not sure how to distinguish, for example, temperature of the gas, ambient and reference temperature. Does reference temperature refer to the gas or ambient temperature?
Source parameters:

- Gas Type
- Gas Temperature
- Gas Pressure
- Gas Relative Humidity
- Gas Compressibility
- Gas Velocity

Reference parameters

- Reference Temperature
- Reference Pressure
- Reference Relative Humidity
- Reference Compressibility

Ambient conditions:

- Ambient Temperature
- Ambient Pressure
- Ambient Relative Humidity

Other parameters

- Outlet Pressure (does this refer to UUT output pressure or the source, how is this different than gas pressure above?)
- Reynolds Number (is this parameter value taken as a given property of the gas? Or a ratio measurand?

**Comment by soorajvenkatesan-Indysoft** [comment]
@MarkKuster @ryanmwhitephd can we go ahead and add this taxa to our database for the further use?

**Comment by ryanmwhitephd** [comment]
Yes, once we have the review from domain expert we will add these.

**Comment by jfedchak** [comment]
We should probably include PipeSize as a optional parameter.
